packages/react/modules/components/toggle-feature/toggle-feature.ts

```diff
@@ -32,8 +31,13 @@ const ToggleFeature = (props: Props): React.ReactNode => {
     if (props.toggledComponent)
       return React.createElement(props.toggledComponent);
 
-    if (props.children && !isEmptyChildren(props.children))
-      return React.Children.only(props.children);
+    if (props.children) {
+      if (typeof props.children === 'function')
+        return props.children({
```

Comment:
Maybe I got this wrong, but I think we should render here the children as a function, in case it's a function.

Reply:
This happens in L45 if I am not mistaken?

Reply:
In both yes. But we need to duplicate it as otherwise we might not get to L45 (for example if the render prop is also defined).
Unless you don't bother about this case.